drafts/use-cases/5.creating-new-event.md

```diff
@@ -22,25 +23,26 @@ var client = new HydraClient();
 var operation = client.get("http://example.com")
     .getApiDocumentation()
     .getEntryPoint()
-    .getOperations()
-    .filter(op => op.method == "POST" && op.expects == "Event")[0];
+    .onLink('#events')
```

Comment:
Then I'd agree with @tpluscode about renaming it to `getLink`. I thini `onLink` is somehow related to events.